### PR TITLE
Support new emojiReaction data format

### PR DESCRIPTION
**Pull request by tgolen**
- [x] Held on https://github.com/Expensify/Auth/pull/7811

### Details
In order to facilitate the use of `Onyx.merge()` when adding and removing emoji, the data formation of `reactions` was changed from an array to `emojiReactions` which is an object. This allows for much better incremental changes of the emoji data so that the entire array of emojis doesn't have to be passed to `Onyx.set()` which is known for causing race conditions (explained in detail [here](https://github.com/Expensify/Web-Expensify/pull/37193#issuecomment-1540636005)).

This leads to a lot of cascading changes for how emojis are parsed, displayed, and modified, which this PR is updating.

Lastly, there is a network param called `useEmojiReactions` which allows the PHP and Auth to maintain backwards compatibility to support both `reactions` and `emojiReactions` at the same time while the front-end code is being migrated.

Old Format:
```
"reactionName": {
          emoji: string,
          users: {
              accountID: string,
              skinTone: number,
          }[]
      }
```

New Format:
```
{+1: 
    {
        createdAt: "2023-07-06 19:24:43"
        emojiName: "+1"
        users: 
            273: {
                skinTones: {-1: '2023-07-06 19:24:43'}
            } 
            274: null
    }
}
```

### Fixed Issues
$ https://github.com/Expensify/App/issues/19531

### Tests
1. Sign into NewDot with User A
2. Sign into NewDot with User B
3. Have each of them open a chat with each other
4. Have them both adding and removing reactions from messages
5. Verify that the emojis are added and removed as you would expect them to

- [ ] Verify that no errors appear in the JS console

### Offline tests
1. Go offline
2. Add a reaction to a message
3. Come back online and wait a few seconds (to ensure the network requests are all done)
4. Refresh the page
5. Verify the reaction is still there

### QA Steps
Same as the tests

- [ ] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I ran the tests on **all platforms** & verified they passed on:
    - [ ] Android / native
    - [ ] Android / Chrome
    - [ ] iOS / native
    - [ ] iOS / Safari
    - [ ] MacOS / Chrome / Safari
    - [ ] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

</details>

<details>
<summary>Mobile Web - Chrome</summary>

</details>

<details>
<summary>Mobile Web - Safari</summary>

</details>

<details>
<summary>Desktop</summary>

</details>

<details>
<summary>iOS</summary>

</details>

<details>
<summary>Android</summary>

</details>

### CODE REVIEW

**Comment by tgolen** [comment]
TODOs:
- [ ] Document new proptypes
- [ ] Remove old methods for adding and removing emoji
- [ ] Fix a console error that happens when adding the first quick emoji to a message
- [ ] Test test test

**Comment by tgolen** [comment]
This has a HOLD on it, but it's OK to start reviewing it. The code is fully tested with the Web-E PR and is in its final state.

File: src/components/Reactions/ReportActionItemEmojiReactions.js
Comment (MariaHCD) [inline src/components/Reactions/ReportActionItemEmojiReactions.js:65]: If we introduced millisecond precision to the created timestamp for emoji reactions, I think it would solve this issue?

File: src/libs/actions/Report.js
Comment (MariaHCD) [inline src/libs/actions/Report.js:1457]: Related to the previous comment, is there a way to format the createdAt with ms precision like we do for the [lastRead timestamp](https://github.com/Expensify/Auth/blob/fde937689b7ddd7747a41553fb44b19c45224938/auth/lib/DB.cpp#L11)?

Also, why are we sending createdAt in the frontend, when we can generate this timestamp in Auth?

File: src/components/Reactions/ReportActionItemEmojiReactions.js
Comment (tgolen) [reply src/components/Reactions/ReportActionItemEmojiReactions.js:65]: I think adding ms precision makes it more unlikely, but does not solve this once and for all. This method will solve it for sure though.

File: src/libs/actions/Report.js
Comment (tgolen) [reply src/libs/actions/Report.js:1457]: We are creating it frontend so that it works optimistically and offline.

**Review by rushatgabhane — CHANGES_REQUESTED** [review]

File: src/components/Reactions/ReportActionItemEmojiReactions.js
Comment (tgolen) [reply src/components/Reactions/ReportActionItemEmojiReactions.js:83]: Ah, I think this is probably still needed. I don't know how it got removed.

**Comment by tgolen** [comment]
OK, PR is updated with:

- More simple Onyx key without the reportID (means you need to sign out and sign back in to get the proper Onyx data)
- Add some comments to explain what will get cleaned up
- Renamed a method that is a callback
- Used DRYed proptypes
- Re-added a mistakingly removed prop for tooltip content keys
- Tried another attempt at sorting the emojis in a more reliable way for when emojis are removed

**Review by stitesExpensify — COMMENTED** [review]
Code is looking great!

File: src/components/Reactions/MiniQuickEmojiReactions.js
Comment (tgolen) [reply src/components/Reactions/MiniQuickEmojiReactions.js]: Good catch. This should be `emojiReactions`

File: src/libs/actions/Report.js
Comment (tgolen) [reply src/libs/actions/Report.js]: I looked up the official JSDocs, and this style isn't mentioned, so I changed it to be a more accepted style.

**Comment by tgolen** [comment]
Updated branch:
- Corrected a prop reference that was passing `undefined`
- Corrected some JSDoc params

**Review by MariaHCD — COMMENTED** [review]
LGTM. There are some merge conflicts, though!

**Review by neil-marcellini — DISMISSED** [review]
Wow, lot's of changes for this. It tests well and looks good. I left a few NAB comments.

**Comment by neil-marcellini** [comment]
Also, I suppose we should update the description to say that we're holding on https://github.com/Expensify/Web-Expensify/pull/37193.

**Comment by neil-marcellini** [comment]
@stitesExpensify we can remove the hold, fix the merge conflicts, and merge this I think. Please lmk when you're ready for another review.

**Comment by stitesExpensify** [comment]
@neil-marcellini I don't think so. The PHP part was reverted last week because it broke staging. We need [this](https://github.com/Expensify/App/pull/20087) to be merged and deployed, and then re-merge the PHP, and then we can do this

**Comment by stitesExpensify** [comment]
Lots of conflicts. I'll be fixing them this week.

**Comment by stitesExpensify** [comment]
FYI these conflicts are _hard_ because of the transition to functional components. I'm still working through them.

**Comment by neil-marcellini** [comment]
I'm marking this as a draft so I can easily ignore it until it's ready. I'll review as soon as it is ready.

File: src/pages/home/report/ReportActionItem.js
Comment (stitesExpensify) [reply src/pages/home/report/ReportActionItem.js:208]: Good question. Since it was already there before this PR, let's just leave it for now

**Comment by stitesExpensify** [comment]
Hi @rushatgabhane @neil-marcellini @MariaHCD this is finally off hold and I have fixed a ton of merge conflicts. If you could prioritize reviewing this so that it doesn't get super conflicted I would greatly appreciate it :bow:

File: src/libs/actions/Report.js
Comment (MariaHCD) [inline src/libs/actions/Report.js:1497]: skinTone can be `0`, right? So this condition will evaluate to true for that skinTone?

File: src/pages/home/report/ContextMenu/ContextMenuActions.js
Comment (MariaHCD) [reply src/pages/home/report/ContextMenu/ContextMenuActions.js:60]: I think the existing function name is a bit more readable/intuitive. But I don't feel too strongly about this.

File: src/pages/home/report/ReportActionItem.js
Comment (MariaHCD) [inline src/pages/home/report/ReportActionItem.js:206]: I don't think we need to call `toggleEmojiReaction` twice here?

File: src/components/AttachmentCarousel/index.js
Comment (neil-marcellini) [inline src/components/AttachmentCarousel/index.js]: NAB: This is against the convention in our style guide https://github.com/Expensify/App/blob/ac3ac1f1257d3224fe206030e66cb89bacf46910/contributingGuides/STYLE.md#L165

File: src/components/Reactions/ReportActionItemEmojiReactions.js
Comment (neil-marcellini) [reply src/components/Reactions/ReportActionItemEmojiReactions.js:81]: `user.skinTones` sounds like an array or object. @stitesExpensify would you please add an example of the old and new emoji reactions formats to the PR description? I have forgotten what the formats look like.

File: src/components/Reactions/ReportActionItemEmojiReactions.js
Comment (neil-marcellini) [reply src/components/Reactions/ReportActionItemEmojiReactions.js:93]: +1 reactionUserAccountIDs is much more clear.

File: src/libs/actions/Report.js
Comment (neil-marcellini) [inline src/libs/actions/Report.js:1599]: Apparently this doesn't work for the first message in a threads, but we have [another issue](https://github.com/Expensify/App/issues/22235) to fix it so it's ok for now.

File: src/pages/home/report/ContextMenu/ContextMenuActions.js
Comment (neil-marcellini) [reply src/pages/home/report/ContextMenu/ContextMenuActions.js:60]: I agree with Rushat, let's follow our style guide.

**Review by neil-marcellini — CHANGES_REQUESTED** [review]
Looking great, just a few small changes. I'll also wait until Rajat tests and approves before I do a final review.

File: src/components/Reactions/ReportActionItemEmojiReactions.js
Comment (stitesExpensify) [reply src/components/Reactions/ReportActionItemEmojiReactions.js:81]: Updated the OP with the new and old format. I believe you're right that we can never not have a skintone, it will always have the object or be null, so we can just always count size here

File: src/libs/actions/Report.js
Comment (stitesExpensify) [reply src/libs/actions/Report.js:1497]: It can, nice catch!

Comment (stitesExpensify) [reply src/libs/actions/Report.js:1497]: Updating it to `_.isUndefined`

File: src/pages/home/report/ReportActionItem.js
Comment (stitesExpensify) [reply src/pages/home/report/ReportActionItem.js:206]: Correct. Bad merge

**Comment by stitesExpensify** [comment]
Okay I fixed all of the changes requested, however I found a really weird bug that I'm not sure how to fix. When you react with `skinTone=0` (the darkest skin tone) the skinTones object is coming back as an array 😬 I think that javascript is somehow converting the object `{0: asdf}` into an array because it is 0 indexed...

**Comment by stitesExpensify** [comment]
Or maybe it's even on the PHP side???

**Comment by stitesExpensify** [comment]
I figured out _why_ it's happening, but not how to fix it :( Posting in #engineering-chat

**Comment by stitesExpensify** [comment]
https://expensify.slack.com/archives/C03TQ48KC/p1688678586327129

json_encode is our problem. We're going to need a PHP PR before we can do this :(

**Comment by stitesExpensify** [comment]
Okay I think I have a solution here https://github.com/Expensify/Web-Expensify/pull/38090

**Comment by stitesExpensify** [comment]
Solution for that problem was merged. We will need for it to go to prod before this can be merged

**Comment by stitesExpensify** [comment]
Okay the Fix for the `skinTone=0` issue is now on production, so this is ready for yet another review! I did another round of big merge conflicts because we now support spanish, so any additional testing would be appreciated. We're so close!

**Review by MariaHCD — COMMENTED** [review]
Looks good overall! Just two minor comments. Did some light testing on web and seems to work as expected 🎊

File: src/pages/home/report/ContextMenu/ContextMenuActions.js
Comment (stitesExpensify) [reply src/pages/home/report/ContextMenu/ContextMenuActions.js]: Nope you're right, it was unused!

**Comment by stitesExpensify** [comment]
Updated!

**Review by MariaHCD — APPROVED** [review]
LGTM!

**Comment by stitesExpensify** [comment]
Aweseome! @rushatgabhane @neil-marcellini if you could check this out before there are more conflicts I would really appreciate it :)

**Review by neil-marcellini — APPROVED** [review]
Looks good to me assuming it tests well.

**Comment by stitesExpensify** [comment]
Bump @rushatgabhane you're the last reviewer 😄

**Comment by rushatgabhane** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android / native
    - [ ] Android / Chrome
    - [ ] iOS / native
    - [ ] iOS / Safari
    - [ ] MacOS / Chrome / Safari
    - [ ] MacOS / Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>



</details>

<details>
<summary>Mobile Web - Chrome</summary>



</details>

<details>
<summary>Mobile Web - Safari</summary>



</details>

<details>
<summary>Desktop</summary>



</details>

<details>
<summary>iOS</summary>



</details>

<details>
<summary>Android</summary>



</details>

**Review by rushatgabhane — CHANGES_REQUESTED** [review]
Bug: can't react to the first message in a thread.
Steps:
1. Send a message 
2. Click reply in thread
3. React to the message

Expected: Can react to the first (parent) message of a thread
Actual: can't react


https://github.com/Expensify/App/assets/29673073/9df0bd7b-a3bf-4b18-8496-f7a5a01d52fe

**Comment by rushatgabhane** [comment]
@aimane-chnaif thanks for the info!

**Review by rushatgabhane — APPROVED** [review]
LGTM too

**Comment by ayazhussain79** [comment]
Is that regression of this PR?
"0:undefined:" displayed in reaction emoji tooltip and some time  "0::"
Bug posted on salack https://expensify.slack.com/archives/C049HHMV9SM/p1689854537784799



Screenshot 2023-07-20 at 5 02 49 PM



https://github.com/Expensify/App/assets/56740856/b9cbdf77-8070-4eea-8246-18f48eefeec1

**Pull request merged**
